dll/cpl/sysdm/general.c

```diff
@@ -201,10 +206,12 @@ LRESULT CALLBACK RosImageProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam
                     AlphaBlend(hCreditsDC, 0, 0, pImgInfo->cxSource, pImgInfo->cySource, hLogoDC, 0,  0, pImgInfo->cxSource, pImgInfo->cySource, BlendFunc);
                     AlphaBlend(hCreditsDC, 0, offset, pImgInfo->cxSource, pImgInfo->cySource, hLogoDC, 0,  0, pImgInfo->cxSource, pImgInfo->cySource, BlendFunc);
 
-                    DeleteDC(hLogoDC);
-                    DeleteDC(hCreditsDC);
-
                     timerid = SetTimer(hwnd, 1, ANIM_TIME, NULL);
+
+Cleanup:
```

Comment:
We are leaking the hFont.

Reply:
This is not the scope of the PR, but I, i take this fix by opportunity